validator/client/validator_metrics.go

```diff
@@ -33,28 +33,15 @@ func (v *validator) LogValidatorGainsAndLosses(ctx context.Context, slot uint64)
 		return err
 	}
 
-	log.WithFields(logrus.Fields{
```

Comment:
can we at least give the average validator performance as a log ? we dont have to remove the whole chunk

Reply:
Most of these are not very useful, not sure what validators would get from avg validator eth balance 😄